### DISCUSSION TITLE
stop imposing MFA on your users

### DISCUSSION THREAD

**Discussion by YievCkim** [community/community#134284] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

MFA slows down workflow. You have to have your cell phone with you at all times. My repo is already protected by an SSH key, so please stop it.

**Comment by DarshanAnand007** [comment]
While SSH keys provide strong security, MFA adds an essential extra layer of protection against potential threats. It helps ensure that even if your SSH key is compromised, your account remains secure , dont you think

**Reply by YievCkim** [reply]
What threat? GH doesn't host financial or banking services it's a platform to host open source project. If some companies want to use GH and protect their code with MFA It's their choice. But I don't see the point to impose that to all user. Putting MFA everywhere means making life impossible for all Internet users. Find something else.

**Reply by DarshanAnand007** [reply]
yes i aggree :)

**Reply by YievCkim** [reply]
Maybe something more adaptative would be a good trade off. By example MFA could be asked when suspicious action is detected (with algorithms or AI) or when a PR is made on a repo owned by another account.

**Reply by DarshanAnand007** [reply]
Yeah but eod it also detepend on how many people will use it

**Reply by DarshanAnand007** [reply]
and we can close the discussion ?

**Reply by YievCkim** [reply]
I would like to know the kind of threat MFA protect to. Give me some examples.
